src/shared/containers/challenge-listing/Listing/index.jsx

```diff
@@ -627,89 +627,91 @@ export class ListingContainer extends React.Component {
+          <MetaTags
+            description={description}
+            image={ogImage}
+            siteName="Topcoder"
+            title={title}
+          />
+          {banner}
+          <ChallengeListing
+            location={location}
+            history={history}
+            activeBucket={activeBucket}
+            challenges={challenges}
+            openForRegistrationChallenges={openForRegistrationChallenges}
+            myChallenges={myChallenges}
+            myPastChallenges={myPastChallenges}
+            allChallenges={allChallenges}
+            pastChallenges={pastChallenges}
+            challengeTypes={challengeTypes}
+            challengesUrl={challengesUrl}
+            communityFilter={communityFilter}
+            communityName={communityName}
+            defaultCommunityId={defaultCommunityId}
+            expanding={expanding}
+            expandedTags={expandedTags}
+            expandTag={expandTag}
+            // extraBucket={extraBucket}
+            filterState={filter}
+            hideSrm={hideSrm}
+            // hideTcLinksInFooter={hideTcLinksInSidebarFooter}
+            keepPastPlaceholders={keepPastPlaceholders}
+            // lastUpdateOfActiveChallenges={lastUpdateOfActiveChallenges}
+            // eslint-disable-next-line max-len
+            needLoad={needLoad}
+            loadingMyChallenges={Boolean(loadingMyChallengesUUID)}
+            loadingMyPastChallenges={Boolean(loadingMyPastChallengesUUID)}
+            loadingAllChallenges={Boolean(loadingAllChallengesUUID)}
+            loadingOpenForRegistrationChallenges={Boolean(loadingOpenForRegistrationChallengesUUID)}
+            loadingOnGoingChallenges={Boolean(loadingActiveChallengesUUID)}
+            // eslint-disable-next-line max-len
+            // loadingChallenges={Boolean(loadingActiveChallengesUUID) && Boolean(loadingOpenForRegistrationChallengesUUID) && Boolean(loadingMyChallengesUUID)}
+            loadingPastChallenges={Boolean(loadingPastChallengesUUID)}
+            loadingReviewOpportunities={Boolean(loadingReviewOpportunitiesUUID)}
+            newChallengeDetails={newChallengeDetails}
+            openChallengesInNewTabs={openChallengesInNewTabs}
+            preListingMsg={preListingMsg}
+            prizeMode={prizeMode}
+            selectBucket={selectBucket}
+            selectChallengeDetailsTab={selectChallengeDetailsTab}
+            selectedCommunityId={selectedCommunityId}
+            loadMorePast={loadMorePast}
+            loadMoreReviewOpportunities={loadMoreReviewOpportunities}
+            loadMoreMy={loadMoreMy}
+            loadMoreMyPast={loadMoreMyPast}
+            loadMoreAll={loadMoreAll}
+            loadMoreOpenForRegistration={loadMoreOpenForRegistration}
+            loadMoreOnGoing={loadMoreOnGoing}
+            reviewOpportunities={reviewOpportunities}
+            setFilterState={(state) => {
+              setFilter(state);
+              // if (activeBucket === BUCKETS.SAVED_FILTER) {
```

Comment:
Why not remove all commented-out lines here?

Reply:
Fixed in next commit